src/views/partials/comment-html.handlebars

```diff
@@ -0,0 +1,12 @@
+<h2>Comments</h2>
+{{#each comments}}
+<h3>{{user.initials}}</h3>
+<p class="comment">{{comment}}</p>
+{{/each}}
+<form id="comment-form" action="/boards/card/comments/{{boardId}}/{{cardId}}" method="PUT">
+     <label>
```

Comment:
Same thing, not sure if it needs a for attribute but that's how we did it in class

Reply:
He showed us both 👍